core/flyout_horizontal.js

```diff
@@ -275,6 +321,22 @@ Blockly.HorizontalFlyout.prototype.wheel_ = function(e) {
 };
 
 /**
+ * Select a category using the scroll position
+ * @param  {number} pos The scroll position.
+ */
+Blockly.HorizontalFlyout.prototype.selectCategoryByScrollPosition = function(pos) {
```

Comment:
This function is the same for horizontal and vertical. Move it to flyout_base, and mark it package unless you're planning to use it externally.

Reply:
nice- done!